src/views/RecIM/views/Activity/components/ScheduleList/index.jsx

```diff
@@ -29,16 +31,26 @@ const ScheduleList = ({ isAdmin, series, activityID, reload, setReload }) => {
   };
 
   // edit button
-  const handleEdit = () => {
+  const handleEditSeries = () => {
     setOpenEditSeriesForm(true);
     handleClose();
   };
 
+  const handleSeriesSchedule = () => {
+    setOpenSeriesScheduleForm(true);
+    handleClose();
+  };
+
   const handleEditSeriesForm = (status) => {
     setReload((prev) => !prev);
```

Comment:
This is a really good point. However, since we fetch the entire pages information in one call, we cannot just load a single piece (eager loading). The only way we could accomplish this is to make a new get route, but that does not make sense to have the data in the eager load and the other get.